app/profileMigration.server.ts

```diff
@@ -0,0 +1,31 @@
+import type { PrismaClient } from "@prisma/client"
+import { getActiveProfiles } from "./lake.server";
+import { consolidateProfilesByEmail } from "./models/profile.server";
+import type { Prisma } from "@prisma/client";
+
+async function getMappedProfiles(): Promise<Prisma.ProfilesCreateInput[]>{
+  const activeProfiles = await getActiveProfiles();
+  const mappedProfiles = activeProfiles.map(lakeProfile => {
```

Comment:
I don't fully like that we are holding the information of all wizeliners in memory, instead of processing one by one. This could potentially be a problem, but very late in the future?

What do you think? Premature optimization?

Reply:
As soon as we query all of the "not terminated" we are adding them all to memory. If i am understanding well you would like to me to try another way to query the information?
If so  i think we could paginate but i did not find any way to do this with the lake dataset that we can get.

Reply:
On the other side, this script is for migration and will be run only once.